test/functional/disablehot.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
```

Comment:
Consider running a linter like flake8 over python modules to check for common nits.